chat-plugins/thestudio.js

```diff
@@ -257,11 +257,37 @@ let commands = {
 		"/aotd quote - View the current Artist Quote of the Day.",
 		"/aotd quote [quote] - Set the Artist Quote of the Day. Requires: # & ~",
 	],
+	song: function (target, room, user) {
+		if (room.id !== 'thestudio') return this.errorReply('This command can only be used in The Studio.');
+		if (!target) return this.parse('/help aotd song');
+		if (!room.chatRoomData || !this.can('mute', null, room)) return false;
+		if (!this.canTalk()) return;
+
+		if (target === 'off' || target === 'disable') {
+			room.chatRoomData.songOfTheDay = false;
+			Rooms.global.writeChatRoomData();
+			return this.privateModCommand(`(${user.name} has disabled the Song of the Day.)`);
+		} else if (target.includes(',')) {
+			let title = target.split(',')[0].trim();
+			let link = target.split(',')[1].trim();
+
+			room.chatRoomData.songOfTheDay = Chat.html`<a href="${link}" title="${title}">${title}</a>.`;
```

Comment:
This should probably be called `songOfTheDayHTML` if it contains HTML, as it does.